### DISCUSSION TITLE
Possibility of Using Fixed Version of WebView2

### DISCUSSION THREAD

**Discussion by impubes** [r0x0r/pywebview#896] [Q&A]
Hi, 
is it possible to use fixed version (which can be downloaded from https://developer.microsoft.com/en-us/microsoft-edge/webview2/ as cab package) of WebView2 in pywebview app?
Thanks 🙏

**Comment by r0x0r** [comment]
I am under impression that webview2 is evergreen and is updated automatically to a new version.
